.eslintrc.js

```diff
@@ -25,76 +25,82 @@ module.exports = {
     'plugin:react/recommended',
   ],
 
-  plugins: [
-    'babel',
-    'react',
-    'json',
-    'import',
-  ],
+  plugins: ['babel', 'react', 'json', 'import', 'prettier'],
 
   globals: {
-    '$': 'readonly',
+    $: 'readonly',
     document: 'readonly',
     QUnit: 'readonly',
     window: 'readonly',
   },
 
   rules: {
+    'prettier/prettier': 'error',
+    'space-before-function-paren': [
+      'error',
+      { anonymous: 'always', named: 'never' },
+    ],
+    indent: 'off',
     'arrow-parens': 'error',
     'no-tabs': 'error',
-    'no-mixed-operators': 'error',
+    'no-mixed-operators': 'off',
+    'no-confusing-arrow': 'off',
```

Comment:
Is there any way these could remain enabled?

Reply:
https://github.com/prettier/eslint-config-prettier/issues/17

Reply:
also i posted some screenies from no-mixed-operators discussion.